### ISSUE TITLE
Round 3, Reviewer 6

### ISSUE THREAD

**Issue by labarba** [barbagroup/bempp_exafmm_paper#21]
The authors addressed most of my minor concerns but failed to address my major critique simply because they do not know how to produce a molecular dynamics (MD) trajectory for testing. It is in within their reach to set up and collect a MD trajectory at least for one of the two tested proteins to showcase the versatility of their computational resource as trajectory processing is a major application for PB solvers.

Considering this and their pushbacks on other reviewers’ comments asking validation of their method on a larger set of real molecules, this reviewer’s enthusiasm for the manuscript is diminished. As it stands, the manuscript appears to be just one reporting preliminary efforts to develop a PB solver. Unfortunately, even for the narrower scope, the authors cannot show that their method agree with other widely used methods under identical testing conditions.

Specifically, in subsection “Performance comparison with APBS”, the limiting values between APBS and their method is too large: ~320kcal/mol out of ~10650kcal/mol. The ~3% difference cannot be ignored in comparison of limiting values given identical parameters (coordinates, charges, & radii) are used. It is suggested that the authors use the same surface generation routine during their analysis in this and additional real molecules to make sure the two methods do agree.

Another troublesome point can be found in subsection “Performance study with direct and derivative formulations”, where the authors spend considerate efforts to discuss an unphysical scenario, 100 random charges inside a sphere of 1 Angstrom. It is unclear why the authors address such a totally unrelated scenario to computational structural biology.

In summary, it is unacceptable that the authors cannot work on additional molecules or MD trajectories as suggested. These extra data points can be used to support the validity of their implementation of a new PB solver. Researchers with working knowledge in computational structural biology can easily get these done.

**Comment by labarba** [comment]
## Reviewer #6

**Comment 1:**
>The authors addressed most of my minor concerns but failed to address my major critique simply because they do not know how to produce a molecular dynamics (MD) trajectory for testing. It is in within their reach to set up and collect a MD trajectory at least for one of the two tested proteins to showcase the versatility of their computational resource as trajectory processing is a major application for PB solvers.

*Reply:*
Although using our software with an MD trajectory (equivalent to an MMPBSA approach) is interesting, and could be an example application of our workflow. The new revision of the paper added substantial new results (binding energy calculations). Extending the work further to another application could be material for a new paper. This article's central point is not aided by extending to yet another application.

**Comment 2:**
>Considering this and their pushbacks on other reviewers’ comments asking validation of their method on a larger set of real molecules, this reviewer’s enthusiasm for the manuscript is diminished. As it stands, the manuscript appears to be just one reporting preliminary efforts to develop a PB solver. Unfortunately, even for the narrower scope, the authors cannot show that their method agree with other widely used methods under identical testing conditions.

*Reply:*
See replies to Reviewer 3. In this revision, we computed binding energies for 9 Barnase-Barstar complexes and demonstrated that our results show agreement with the results from other PB software reported in the literature. See section "Binding energy calculations".

**Comment 3:**
>Specifically, in subsection “Performance comparison with APBS”, the limiting values between APBS and their method is too large: ~320kcal/mol out of ~10650kcal/mol. The ~3% difference cannot be ignored in comparison of limiting values given identical parameters (coordinates, charges, & radii) are used. It is suggested that the authors use the same surface generation routine during their analysis in this and additional real molecules to make sure the two methods do agree.

*Reply:*
See replies to Reviewer 3.

**Comment 4:**
>Another troublesome point can be found in subsection “Performance study with direct and derivative formulations”, where the authors spend considerate efforts to discuss an unphysical scenario, 100 random charges inside a sphere of 1 Angstrom. It is unclear why the authors address such a totally unrelated scenario to computational structural biology.

*Reply:*
See replies to Reviewer 3

**Comment 5:**
>In summary, it is unacceptable that the authors cannot work on additional molecules or MD trajectories as suggested. These extra data points can be used to support the validity of their implementation of a new PB solver. Researchers with working knowledge in computational structural biology can easily get these done.

*Reply:*
In reply to Comment 1.
